challenge.rb

```diff
@@ -45,10 +55,19 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working == true && (current_temp > desired_temp)
+    return "Turn on the AC."
+  elsif ac_working == true && (current_temp <= desired_temp)
+    return "Just right!"
+  elsif ac_working == false && (current_temp > desired_temp)
+    return "Fix the AC now! It's hot!"
+  elsif ac_working == false && (current_temp <= desired_temp)
+    return "Fix the AC whenever you have the chance. It's cool."
+  end
```

Comment:
Correct, but see my above comment about redundancy. You can just write `ac_working` or `!ac_working` to indicate whether the value of `ac_working` is `true` or `false`.